br/pkg/lightning/config/config_test.go

```diff
@@ -1330,6 +1330,8 @@ func TestAdjustBlockSize(t *testing.T) {
 	cfg.TikvImporter.BlockSize = 0
 
 	err := cfg.Adjust(context.Background())
+	// should return "dial tcp 127.0.0.1:10080: connect: connection refused" error,
```

Comment:
Sorry I think I did not write this test correctly before, the `dial tcp 127.0.0.1:10080: connect: connection refused` error has been tested by `TestAdjustConnectRefused`.

Maybe we can add
```
ts, host, port := startMockServer(t, http.StatusOK,
	`{"port":6666,"advertise-address":"121.212.121.212:5555","path":"34.34.34.34:3434"}`,
)
defer ts.Close()
```
```
cfg.TiDB.Host = host
cfg.TiDB.StatusPort = port
``` 
in this test, and change to `require.NoError(t, err)`?

Reply:
we can do it in later pr, ok for now